File: src/main/java/com/ideabaker/samples/springsecurity/repository/MovieRepository.java
package com.ideabaker.samples.springsecurity.repository;

import com.ideabaker.samples.springsecurity.model.Movie;
import org.springframework.data.mongodb.repository.ReactiveMongoRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface MovieRepository extends ReactiveMongoRepository<Movie, Long> {
}

File: src/main/kotlin/com.ideabaker.samples.springsecurity/domain/MovieRepository.kt
package com.ideabaker.samples.springsecurity.domain

import org.springframework.data.mongodb.repository.ReactiveMongoRepository
import org.springframework.stereotype.Repository

/**
 *
 * @author Arthur Kazemi<bidadh@gmail.com>
 * @since 2018-12-29 10:50
 */
@Repository
interface MovieRepository : ReactiveMongoRepository<Movie, Long>
